src/models/user.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const userSchema = Schema({
+  userName: {
+    type: String,
+    required: [true, 'Username is required'],
```

Comment:
Making usernames unique would be a better idea to allow sign in with email and username.

Reply:
I changed it. Thanks for the feedback.